laiaaa0/assignment3/Dictionary.cpp

```diff
@@ -1,47 +1,45 @@
 #include "Dictionary.h"
 
-bool Dictionary::IsPrefixOfWord(std::string prefix, std::string word){
-  if (prefix.size()>word.size()) return false;
-  else {
-    for (int i=0; i<prefix.size(); ++i){
-      if (word[i]!=prefix[i]) return false;
-    }
-    return true;
+bool Dictionary::IsPrefixOfWord(std::string prefix, std::string word) const{
```

Comment:
As Flavio mentioned, string has a compare function that can be used to simplify this, a lot :)
You can basically do:
`return word.compare(0, prefix.size(), prefix) == 0;`